utils/githooks/commit-msg.d/10-watermark.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+"""Add watermark to commit message"""
+import os
+import sys
+
+
+def check_if_run(name):
+    """Check existence of a file, removes file"""
+    if not os.path.exists(f".{name}"):
+        print(f"Required githook {name} is not installed. See utils/githooks/README.md")
+        return False
+    os.remove(f".{name}")
+    return True
+
+
+def find_hooks():
+    """Find the required git hooks"""
+    hooks = []
+    for fname in os.listdir('utils/githooks'):
+        if fname == "commit-msg.d":
+            continue
+        if not fname.endswith('.d'):
+            continue
+        if not os.path.isdir(os.path.join("utils/githooks", fname)):
+            continue
+        hooks.append(fname[:-2])
+    return hooks
+
+
+def run_check():
+    """Run the checks for the required commit hooks"""
+    empty_commit = True
+    msg = []
+    with open(sys.argv[1], "r") as commit_msg:
+        msg = commit_msg.readlines()
+        for line in msg:
+            if line.startswith("Required-githooks: true"):
+                sys.exit(0)
+            elif line.startswith("Signed-off-by"):
+                continue
+            elif line.strip() == "":
+                continue
+            elif line.startswith("# ------------------------ >8 ------------------------"):
```

Comment:
when I did git commit -a -s -v, it shows a diff at the bottom and this line seems to be the one that turns off everything below